tests/app/helpers/test_template_helpers.py

```diff
@@ -813,3 +813,14 @@ def test_include_csrf_token(app: Flask, include_csrf_token: bool):
         ).context["include_csrf_token"]
 
     assert result == include_csrf_token
+
+
+def test_health_survey_config(app: Flask):
+    with app.app_context():
+        survey_config = get_survey_config(
+            theme=SurveyType.HEALTH,
+            schema=QuestionnaireSchema({}),
+            language="en",
+        )
+
+    assert isinstance(survey_config, SocialSurveyConfig)
```

Comment:
This isn't needed. A test to check in instance already existing. You just need to update it https://github.com/ONSdigital/eq-questionnaire-runner/blob/support-health-theme/tests/app/helpers/test_template_helpers.py#L579-L580

Reply:
Oops, I didn't realise we check instance, updated tests now and removed the new one, thanks!